lib/papertrail/connection.rb

```diff
@@ -38,6 +38,10 @@ def initialize(options)
       end
     end
 
+    def search_query
```

Comment:
Good idea to push state management into this class. 👍 I think it makes it clear that `Connection` isn't a great name for this class (I'd expect to be able to execute multiple searches on a single `Connection`) but I don't have a better name.